examples/common/QRCode/BUILD.gn

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+config("qrcode-common_config") {
+  include_dirs = [ "repo/c" ]
+
+  cflags = [ "-Wno-type-limits" ]
```

Comment:
I think you should just set this globally unless we're going to fix it soon. It's pretty confusing that adding a dependency on qrcode relaxes warning flags. It means that some subset of the project including all the examples will not be bound by this warning...